src/module/identifier/model.hpp

```
@@ -0,0 +1,54 @@
+                     std::function<void(std::expected<std::vector<Ball2D>, std::string>)>) noexcept
+        -> void;
+
+    struct AsyncResult final {
+        using handle_type = std::coroutine_handle<>;
+        OpenVinoNet& network;
+        const Image& readonly;
+        std::expected<std::vector<Ball2D>, std::string> result{std::unexpected{"Nothing here"}};
+        auto await_resume() noexcept {
+            return std::move(result);
+        }
+        auto await_suspend(handle_type coroutine) noexcept {
```

Clang (14.0.6)

[warning] parameter 'coroutine' is unused

(misc-unused-parameters)